instrumentation/lettuce/lettuce-5.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/LettuceInstrumentationModule.java

```diff
@@ -15,13 +15,11 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package io.lumigo.javaagent.instrumentation.lettuce.v5_1;
+package io.opentelemetry.javaagent.instrumentation.lettuce.v5_1;
```

Comment:
preserve also the original package folders

Reply:
Done